Setup/KuduSetupCustomWebPiFeed.xml

```diff
@@ -212,4 +208,123 @@
       </installer>
     </installers>
   </entry>
-</feed>
\ No newline at end of file
+  <!-- entry for iisnode locked to a known version -->
+  <entry>
+    <productId>iisnode_0_2_11</productId>

```

Comment:
So the reason for this is that the standard WebPI feed doesn't have those newer entries?